rpc/src/main/scala/org/bitcoins/rpc/RpcUtil.scala

```diff
@@ -17,42 +17,52 @@ trait RpcUtil extends BitcoinSLogger {
 
   /**
    * The returned Future completes when condition becomes true
-   * @param condition The condition being waited on
+   * @param conditionF The condition being waited on
    * @param duration The interval between calls to check condition
    * @param maxTries If condition is tried this many times, the Future fails
    * @param system An ActorSystem to schedule calls to condition
    * @return A Future[Unit] that succeeds if condition becomes true and fails otherwise
    */
-  def retryUntilSatisfied(
-    condition: => Boolean,
+  def retryUntilSatisfiedF(
```

Comment:
I think it wouldn't hurt to also have a `retryUntilSatisfied` that takes in a `=> Boolean` with no `Future` which could literally be

```
def retryUntilSatisfiedF(
  condition: => Boolean,
  duration: FiniteDuration = 100.milliseconds,
  maxTries: Int = 50)(implicit system: ActorSystem): Future[Unit] = {
    retryUntilSatisfiedF(Future.successful(conditionF), duration, maxTries)
}